docs/src/main/asciidoc/getting-started-guide.adoc

```diff
@@ -47,10 +47,16 @@ This guide also covers the testing of the endpoint.
 
 == Solution
 
-We recommend that you follow the instructions in the next sections and create the application step by step.
+We recommend that you follow the instructions from <<bootstrapping-the-project,Boostrapping project>> and onwards to create the application step by step.
+
 However, you can go right to the completed example.
 
-Clone the Git repository: `git clone {quickstarts-clone-url}`, or download an {quickstarts-archive-url}[archive].
+Download an {quickstarts-archive-url}[archive] or clone the git repository:
+
+[source,shell,subs=attributes+]
+----
+git clone {quickstarts-clone-url}
+----
```

Comment:
This section is present in most of the guides? Do you plan to upgrade all of them? Or we just make the getting started one smoother and suppose people looking at the other guides know how to git clone?

Reply:
i've just made it here so far - I can make updates for the others too; but I suggest we do that in a follow up PR.

Reply:
I think we can be very descriptive in the very first guide for beginners and suppose they are able to do a git clone in the others. I don't think it's worth changing all the guides.